engine/src/flutter/lib/web_ui/test/felt_config.yaml

```diff
@@ -98,6 +108,11 @@ run-configs:
     canvaskit-variant: full
 
 test-suites:
+  - name: chrome-dart2js-experimental-webparagraph-ui
```

Comment:
Presumably we don't actually want this running on CI since we aren't building the CanvasKit variant yet, correct? In the skwasm_heavy PR (https://github.com/flutter/flutter/pull/166619) I added an `enable-ci` flag that we parse out that prevents the `felt generate-builder-json` command from including it in the CI configuration. We should do that here before landing, since we don't want people to accidentally add this suite if they have to rerun `felt generate-builder-json`.

Reply:
Ah nice catch!

Reply:
Done